recipes/libassert/all/conandata.yml

```diff
@@ -18,6 +22,10 @@ sources:
     sha256: "e1bb3b50767994ca4d0f60b7977b279cf32b8569ff92c5830e7a1de567b82fd5"
 
 patches:
+  "2.0.0":
+    - patch_file: "patches/2.0.0/0001-fix-export-and-include.patch"
```

Comment:
@jeremy-rifkin Would it be possible to integrate this patch in the upstream?

Reply:
Yes indeed, already done at https://github.com/jeremy-rifkin/libassert/commit/25c1f3e43737ab18490a0d9430cb1c70f976a662.